generated/model_spec_0.3.0.json

```diff
@@ -29,6 +29,67 @@
             ],
             "type": "object"
         },
+        "ImplicitInputShape": {
```

Comment:
I think we can get rid of the auto-generated json schema for now.
As far as I remember our recent discussions we probably don't need it; and so removing it would get rid of another potential source of confusion. (And adding it again in case we need it is cheap)